tests/env/test_ggg.py

```diff
@@ -0,0 +1,25 @@
+"""Tests for PRBench environment provider."""
+
+from programmatic_policy_learning.envs.providers.ggg import ggg
+
+
+class DummyEnvConfig:
```

Comment:
to match what will happen with hydra, I think you should be able to do something like:
```python
from omegaconf import OmegaConf, DictConfig

cfg: DictConfig = OmegaConf.create({
    "env": {
        "make_kwargs": {"id": "TwoPileNim0-v0"}
    }
})
```